ios/RNTwilioVoice/RNTwilioVoice.m

```diff
@@ -347,6 +347,9 @@ - (void)callInviteReceived:(TVOCallInvite *)callInvite {
     if (callInvite.from) {
         from = [callInvite.from stringByReplacingOccurrencesOfString:@"client:" withString:@""];
     }
+    if (callInvite.customParameters[@"CallerName"]) {
```

Comment:
Please use a constant for `CallerName` within this file to avoid copy and paste mistakes in the future

Reply:
Done